trunk/src/graphics/paraview/geom2vtk.F90

```diff
@@ -47,7 +51,12 @@ subroutine geom2vtk(Sname,Sfile, IcE,IcN,IcP)
 !
 !..create list of mdle node numbers,
 !  and count number of visualization points
-   IcP=0; ico=0;
+   IcP=0; ico=0; m = 0;
+
+   if(VIS_VTU) then 
+      allocate(VTU_element_type_offset(NRELES))
```

Comment:
where is this deallocated?

Reply:
Change implemented: deallocated at the end of geom2vtk.F90